modules/fedimint-ln-common/src/gateway_endpoint_constants.rs

```diff
@@ -6,6 +6,7 @@ pub const BALANCE_ENDPOINT: &str = "/balance";
 pub const CONFIGURATION_ENDPOINT: &str = "/config";
 pub const CONNECT_FED_ENDPOINT: &str = "/connect-fed"; // uses `-` for backwards compatibility
 pub const GATEWAY_INFO_ENDPOINT: &str = "/info";
+pub const GATEWAY_MNEMONIC_ENDPOINT: &str = "/mnemonic";
```

Comment:
Nit: Is the `GATEWAY_` prefix necessary here? The file is prefixed with `gateway_` already.

Reply:
Done